forge-gui/res/cardsfolder/m/midnight_crusader_shuttle.txt

```diff
@@ -0,0 +1,14 @@
+Name:Midnight Crusader Shuttle
+ManaCost:4
+Types:Artifact Vehicle
+PT:3/4
+T:Mode$ Attacks | ValidCard$ Card.Self | Execute$ TrigVillainousChoice | TriggerDescription$ Midnight Entity — Whenever CARDNAME attacks, defending player faces a villainous choice — That player sacrifices a creature, or you gain control of a creature of your choice that player controls until end of turn. If you gain control of a creature this way, tap it, and it's attacking that player.
+SVar:TrigVillainousChoice:DB$ VillainousChoice | Defined$ TriggeredDefendingPlayer | Choices$ DBSacrifice,DBGainControl
+SVar:DBSacrifice:DB$ Sacrifice | Amount$ 1 | SacValid$ Creature | Defined$ Remembered | SpellDescription$ That player sacrifices a creature.
+SVar:DBGainControl:DB$ GainControl | Defined$ ChosenCard | Choices$ Creature.RememberedPlayerCtrl | ChoiceTitle$ Select a creature that player controls | NewController$ You | LoseControl$ EOT | SubAbility$ DBTap | SpellDescription$ You gain control of a creature of your choice that player controls until end of turn. If you gain control of a creature this way, tap it, and it's attacking that player.
+SVar:DBTap:DB$ Tap | Defined$ ChosenCard | ConditionDefined$ ChosenCard | ConditionPresent$ Card.YouCtrl | SubAbility$ DBSetAttacking 
+Attacking$ RememberedPlayer | 
```

Comment:
@Hanmac Any idea how this should be scripted ?

Reply:
@tool4ever i think this also needs extra support to make another card Attacking

Reply:
will try to take a look soon :)

Reply:
@Simisays 
you can script it like the most recent Tahngarth version now